contracts/RewardsBooster.sol

```diff
@@ -500,28 +502,26 @@ contract RewardsBooster is Initializable, OwnableUpgradeable, IRewardsBooster {
                     _reportAt <= block.timestamp,
                 'RB010'
             );
+
             // scenario#1: if `disabled` changes from any -> false, by default we consider between lastReportMissedLabor and block.timestamp no misslabor
             // unless specified in _missedLaborChanges
             // scenario#2: if `disabled` changes from any -> true, by default we consider between lastReportMissedLabor and block.timestamp is all misslabored
             // unless specified in _missedLaborChanges
-            uint256 missedLaborAdd;
-            if (_disableds[i]) {
-                missedLaborAdd = _reportAt - runnerDeplReward.lastReportMissedLaborTime;
-            }
+            uint256 maxMissedLabor = _reportAt - runnerDeplReward.lastReportMissedLaborTime;
             if (_missedLaborChanges[i] > 0) {
-                require(
-                    _missedLaborChanges[i] <=
-                        _reportAt - runnerDeplReward.lastReportMissedLaborTime,
-                    'RB011'
-                );
-                missedLaborAdd = _missedLaborChanges[i];
+                require(_missedLaborChanges[i] <= maxMissedLabor, 'RB011');
+            } else {
+                require(!_disableds[i], 'RB012');
```

Comment:
_disableds is the last status of runner, it is to decide missedlabor between lastReport and claim.
_missedLaborChanges is the override.
When _disableds is true and the override is 0, it means it is missed labor all during the report period
when _disableds is false, and override is 0, it means it is healthy all during the report period